index.html

```diff
@@ -254,6 +254,11 @@ <h2>Forum and Contact</h2>
           <div class="well">
 
 	    <h4>News</h4>
+	    <p>
+	      2020/06/30:
```

Comment:
Same here and the lines below

Reply:
The whole file is a mixture of tabs and spaces. I will leave it as is and we replace all of them later.